server/utils/personUtils.ts

```diff
@@ -23,6 +23,14 @@ const isUnknownPerson = (person?: Person): boolean => person?.type === 'UnknownP
 
 const laoName = (person: FullPerson) => (person.isRestricted ? `LAO: ${person.name}` : person.name)
 
+const laoSummaryName = (personSummary: PersonSummary) => {
+  if (personSummary.personType === 'FullPersonSummary') {
+    const { name, isRestricted } = personSummary as FullPersonSummary
+    return isRestricted ? `LAO: ${name}` : name
+  }
+  return personSummary.personType === 'RestrictedPersonSummary' ? 'LAO' : 'Unknown'
+}
```

Comment:
Fair comments.
Yes, the code is duplicated but we have a `Person` type and a `PersonSummary` type which are mirror images.  They contain a `type`/ `personType` respectively which are themselves different union types and select between a [`RestrictedPerson` and `FullPerson`] and [`RestrictedPersonSummary` and `FullPersonSummary`] types. I thought it simpler to duplicate the code rather than try to handle all that type casting shenanigans in one routine.